docs/user-guide/configuration/externalConfig.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: main
+title: External Config
+category: User Guide
+menu: menu
+toc:
+    - title: External Config
+      url: "#external-config"
+
+---
+# External Config
+External config allows a single parent pipeline to create and manage a list of child pipelines with job configuration from parent's `screwdriver.yaml` and source code from itself.
```

Comment:
External config allows a single parent pipeline to create and manage build configuration of other "child" pipelines with a single `screwdriver.yaml`. The source code for each child pipeline will be taken from each child repository.

Reply:
> External config allows a single parent pipeline to create and manage build configuration of other "child" pipelines with a single screwdriver.yaml. The source code for each child pipeline will be taken from each child repository.

I think this sounds better, we need to give a idea that parent pipeline is building source code of the child/external repositories. 

manage build configuration of external source repositories with a single `screwdriver.yaml`